src/auth-sudo.js

```diff
@@ -6,7 +6,8 @@ const LOG_SPARQL_QUERIES = process.env.LOG_SPARQL_QUERIES != undefined ? env.get
 const LOG_SPARQL_UPDATES = process.env.LOG_SPARQL_UPDATES != undefined ? env.get('LOG_SPARQL_UPDATES').asBool() : env.get('LOG_SPARQL_ALL').asBool();
 const DEBUG_AUTH_HEADERS = env.get('DEBUG_AUTH_HEADERS').asBool();
 
-function sudoSparqlClient( extraHeaders = {} ) {
+function sudoSparqlClient( extraHeaders = {}, sparqlEndpoint ) {
```

Comment:
I would personally inline that, but that's just nitpicking
```suggestion
function sudoSparqlClient( extraHeaders = {}, sparqlEndpoint = process.env.MU_SPARQL_ENDPOINT ) {
```